packages/components/pie-cookie-banner/test/accessibility/pie-cookie-banner.spec.ts

```diff
@@ -1,15 +1,10 @@
-
-import { test, expect } from '@justeattakeaway/pie-webc-testing/src/playwright/webc-fixtures.ts';
-import { PieCookieBanner, CookieBannerProps } from '../../src/index.ts';
+import { test, expect } from '@justeattakeaway/pie-webc-testing/src/playwright/playwright-fixtures.ts';
+import { CookieBannerComponent } from '../helpers/page-object/pie-cookie-banner.page.ts';
 
 test.describe('PieCookieBanner - Accessibility tests', () => {
-    test('a11y - should test the PieCookieBanner component WCAG compliance', async ({ makeAxeBuilder, mount }) => {
-        await mount(
-            PieCookieBanner,
-            {
-                props: {} as CookieBannerProps,
-            },
-        );
+    test('a11y - should test the PieCookieBanner component WCAG compliance', async ({ makeAxeBuilder, page }) => {
+        const cookieBannerPage = new CookieBannerComponent(page);
+        await cookieBannerPage.load('');
```

Comment:
Is there not a type error here?
Also feels like we could just have a default empty value for no query strings

Reply:
Oops yeah, clearly missed that one.
Yeah I think we can. I'll make changes now and clean up the rest of the tests.